debian/changelog

```diff
@@ -1,6 +1,8 @@
 matrix-synapse-py3 (1.39.0ubuntu1) UNRELEASED; urgency=medium
 
   * Drop backwards-compatibility code that was required to support Ubuntu Xenial.
+  * Update package triggers so that the virtualenv is correctly rebuild
+    when the system python is rebuilt, on recent Python versions.
```

Comment:
It feels a bit weird to me that this change is listed under 1.39.0 but the PR's target branch isn't release-1.39

Reply:
in the debian changelog, the version number is pretty meaningless while it is UNRELEASED. when we go through the release process, whatever is UNRELEASED gets turned into the version we're releasing.